### ISSUE TITLE
All 'areacello' data failed tests. 

### ISSUE THREAD

**Issue by jbusecke** [leap-stc/cmip6-leap-feedstock#146]
I just noticed when looking for the variable `areacello` that all the legacy stores are in out non-qc catalog. 

I suspect that this is an error in the tests (likely some assumption about time being in the dimensions). 

-[ ] This will require some way to efficiently rerun test over all datasets (except retracted ones) after adjustments have been made to the testing.

**Comment by jbusecke** [comment]
Yeah [this](https://github.com/leap-stc/leap-data-management-utils/blob/ae9acc16b0ff41b0e8401e3244fe637cb7a11fb5/leap_data_management_utils/cmip_testing.py#L45) seems like it would fail if there is no time present in the dataset.

**Comment by briochemc** [comment]
+1 for areacello!

It should be OK to only tested "failed" dataset if a PR fixing this issue only removes tests (skips the time test if there is no time dimension), right? (I'm just asking, I may be confused as I don't really understand much of how this all works 😅)
